yugabyte_cloudformation.yaml

```diff
@@ -35,9 +35,6 @@ Mappings:
     Public2:
       CIDR: "10.0.2.0/24"
 
-Conditions:
-  YBVersion: !Equals [!Ref DBVersion, '1.3.0.0']
```

Comment:
@vp15591 the problem is with this check, this would only be true if the DBVersion is `1.3.0.0`, every other case it would consider as older version even though `1.3.1.0` is a newer version and doesn't need `init_ysql`